cmd/puppeth/module_bootnode.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2017 The go-ethereum Authors
+// This file is part of go-ethereum.
+//
+// go-ethereum is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// go-ethereum is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with go-ethereum. If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/Onther-Tech/plasma-evm/log"
+	"html/template"
+	"math/rand"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+// faucetDockerfile is the Dockerfile required to build a faucet container to
+// grant crypto tokens based on GitHub authentications.
+var bootnodeDockerfile = `
+FROM onthertech/plasma-evm:alltools-latest
```

Comment:
use docker image name from `images["bootnode"]`

Reply:
It updated at https://github.com/Onther-Tech/plasma-evm/commit/dbb5895f5cefd9476bf5953d164ceee85a4b995b